angelssmelting/prototypes/technology/smelting-cobalt.lua

```diff
@@ -9,7 +9,7 @@ data:extend(
       icon_size = 256, icon_mipmaps = 4,
       upgrade = true,
       prerequisites = {
-        "ore-floatation",
+        "ore-leaching",
         "angels-metallurgy-2",
         "angels-coal-processing"
       },
```

Comment:
This tier should now depend on blue science metallurgy rather than green science metallurgy
```suggestion
      prerequisites = {
        "ore-leaching",
        "angels-metallurgy-3",
        "angels-coal-processing"
      },
```